Xamurai/.Docs/Dojo.md

```diff
@@ -4,23 +4,43 @@
 
 ## Instructions﻿
 
-Implement a collection view for a collection of cars (see some of the Experiments pages)
+Implement a collection view for a collection of cars (see some of the Experiments pages). The layout should preferably look like the FlexLayoutPage example, but swiping behavior similar to a Carousel Page where you swipe 2 columns at a time, or like the CollectionView GridLayout example where it snaps into place.
+
+*Hint*: Download the ChartBuilder app on iOS and see how a chart scrolls between pages.
 
 - Use the /Challenges/PagedCollectionPage.xaml for your implementation
 - Implement carousel paging for a 2-column page, but where each element stacks like in the FlexLayoutPage example
 - Use the CarView as the item template
+- All items in a column should be visible (no vertical scrolling necessary)
+- If you collapse a column, the items should shift as in the FlexLayoutPage example.
 - No rules for how you achieve this (there is no single right answer):
 
 For Example, you can choose to use a Carousel View that contains flex layouts by splitting the collection,	or by implementing a snap behavior for a flex layout that scrolls to the next 2 columns on swipe.
 
 - Target Android as primary platform
```

Comment:
@InquisitorJax can we take this out? They could target iOS if they preferred as long as we are using forms we could evaluate either?

Reply:
updated. also moved the guidelines to the top of the file (applies to all challenges)